scripts/i18n.js

```diff
@@ -0,0 +1,58 @@
+let cached = {};
+let langSelect = document.getElementById('langSelect');
+function getLangfile(lang) {
+    return new Promise(function (resolve, reject) {
+        if (cached[lang]) resolve(cached[lang]);
+        axios
+            .get(
+                'https://www.thpatch.net/w/index.php?title=Special:Translate&taction=export&group=themedb&language=' +
```

Comment:
Request will been blocked by CORS. Maybe we can only maintain those json files in local files or Firebase?

![image](https://user-images.githubusercontent.com/7609173/150677243-268afabf-90ce-4811-970f-c8763957adb9.png)

Reply:
yeah if it could be done through firebase that would be ideal

Reply:
I didnt get any CORS issue when running with live server locally though

Reply:
Got it! I will update Firebase & those code to get json files.

Reply:
actually the code in production appears to work fine

Reply:
Wow, but I tried to deploy to GitHub Pages earlier with this branch and still got the same error, maybe a browser limitation? 

I tested in Chrome 97.0.4692.99. Maybe I should deploy again to verify it!

Reply:
Building: https://github.com/Jasonnor/th-music-video-generator/actions/runs/1746537176

Reply:
ah if its the browser specific then it will be better to take it from firebase

Reply:
how is it going on the firebase end?

Reply:
I might be working on it on weekends, it's been a bit busy lately LOL 😄